### PR TITLE
Update HTML file using the workflow

### PR DESCRIPTION
**Pull request by muktoakash**
* **Please check if the PR fulfills these requirements**
- [ x] The commit message follows our contribution guidelines
- [ x] Tests for the changes have been added (for bug fixes / features)
- [ x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Updates the readme file with step-by-step description of how to use this repo to update index.html (or any other html file for that matter). Very useful for automatically updating webpages that are hosted through Github Pages.

* **What is the current behavior?** (You can also link to an open issue here)
Currently all updates happen to README.md of the user's repo. While this is nice, it is even nicer to have the option to automatically update and deploy webpages hosted on Github.

* **What is the new behavior (if this is a feature change)?**
N/A. This just updates the ReadMe file with new instructions.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Repo needs to be forked and action.yml needs to be modified.

* **Other information**:
This is mostly an information update. I had to figure out how to do it myself, so thought others may also benefit from this information.

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
@muktoakash forking the repo is not required in this case. simply pass down the `readme_path` to the main blog post workflow itself. It should work just fine. 

Just use the following file:
```yaml
name: Latest blog post workflow
on:
  schedule:
    # Runs every hour, on the hour
    - cron: "0 * * * *"
  workflow_dispatch:
permissions:
      contents: write

jobs:
  update-readme-with-youtube:
    name: Update this repo's index.html with latest blog titles from blogspot
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      - uses: gautamkrishnar/blog-post-workflow@v1
        with:
          readme_path: 'index.html'
          feed_list: "http://blog_name.blogspot.com/feeds/posts/default?alt=rss"
          template: '<li><a href="$url">$title</a></li>$newline'
          committer_email: "your_email@example.com"
```

File: README.md [hunk 1]
```diff
@@ -243,6 +243,62 @@ by : [Mahdi Qiamast](https://github.com/Qiamast)
...
+      required: false
+  ```
+to
+  ```yml
+   readme_path:
+      description: 'Comma separated paths of the readme files you want to update'
+      default: './index.html' # <-----new
+      required: false
+```
+
```
Comment (gautamkrishnar) [inline README.md:265]: This step is not required

File: README.md [hunk 2]
```diff
@@ -243,6 +243,62 @@ by : [Mahdi Qiamast](https://github.com/Qiamast)
...
+      name: Update this repo's index.html with latest blog titles from blogspot
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v4
+        - uses: username/blog-post-workflow@master
+          with:
+            feed_list: "http://blog_name.blogspot.com/feeds/posts/default?alt=rss"
+            template: '<li><a href="$url">$title</a></li>$newline'
+            committer_email: "your_email@example.com"
+```
```
Comment (gautamkrishnar) [inline README.md:292]: Add an instruction to create index.html

File: README.md [hunk 3]
```diff
@@ -243,6 +243,62 @@ by : [Mahdi Qiamast](https://github.com/Qiamast)
...
+  
+  jobs:
+    update-readme-with-youtube:
+      name: Update this repo's index.html with latest blog titles from blogspot
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v4
+        - uses: username/blog-post-workflow@master
+          with:
+            feed_list: "http://blog_name.blogspot.com/feeds/posts/default?alt=rss"
```
Comment (gautamkrishnar) [inline README.md:289]: add `readme_path: 'index.html'`

**Review by gautamkrishnar — CHANGES_REQUESTED** [review]

**Review by muktoakash — COMMENTED** [review]
@gautamkrishnar Changes made as suggested.

**Comment by gautamkrishnar** [comment]
Thanks for contributing. LGTM 👍🏻

**Comment by gautamkrishnar** [comment]
@all-contributors please add @muktoakash for docs

**Pull request merged**